src/services/importService.ts

```diff
@@ -0,0 +1,186 @@
+            },
+        };
+
+        const targetConnection: IConnection = {
+            id: shortid.generate(),
+            name: "Target Default Name",
+            providerType: "localFileSystemProxy",
+            providerOptions: {
+                folderPath: project.file.path.replace(/[^\/]*$/, ""),
+            },
+        };
+
+        const connections: IConnection[] = [sourceConnection, targetConnection];
+
+        return(connections);
+    }
+
+    /**
+     * Parse v1 project's tag string and return array of ITags
+     * @param project - V1 Project Content and File Information
+     */
+    private parseTags(project: any): ITag[] {
+        const finalTags: ITag[] = [];
+        const tagStrings = project.inputTags.split(",");
+        const tagColors = project.tag_colors;
+
+        for (let i = 0; i < tagColors.length; i++) {
+            const newTag = {
+                name: tagStrings[i],
+                color: tagColors[i],
+            };
+            finalTags.push(newTag);
+        }
+        return finalTags;
+    }
+
+    /**
+     * Generate assets based on V1 Project frames and regions
+     * @param project - V1 Project Content and File Information
+     */
+    private generateAssets(project: any): IAssetMetadata[] {
+        let originalProject: IV1Project;
+        let assetMetadata: IAssetMetadata;
+        const generatedAssetMetadata: IAssetMetadata[] = [];
+        let generatedRegion: IRegion;
+        let assetState: AssetState;
+        const currentTagColorIndex = randomIntInRange(0, TagColors.length);
+
+        originalProject = JSON.parse(project.content);
+
+        for (const frameName in originalProject.frames) {
+            if (originalProject.frames.hasOwnProperty(frameName)) {
+                const frameRegions = originalProject.frames[frameName];
+                assetState = originalProject.visitedFrames.indexOf(frameName) > -1 && frameRegions.length > 0
+                             ? AssetState.Tagged : (originalProject.visitedFrames.indexOf(frameName) > -1
+                             ? AssetState.Visited : AssetState.NotVisited);
+
+                assetMetadata = {
+                    asset: {
+                        id: new MD5().update(frameName).digest("hex"),
+                        type: AssetType.Image,
+                        state: assetState,
+                        name: frameName,
+                        // check on Windows too
+                        path: `file:${project.file.path.replace(/[^\/]*$/, "")}${frameName}`,
+                        size: {
+                            width: frameRegions.length > 0 ? frameRegions[0].width : null,
+                            height: frameRegions.length > 0 ? frameRegions[0].height : null,
+                        },
+                        format: frameName.split(".").pop(),
+                    },
+                    regions: [],
+                };
+
+                for (const region of frameRegions) {
```

Comment:
Also - sync with @tbarlow12  - he may have some helper methods that help convert from v1 regions to v2 regions... since I believe this may be very similar if not the same as the region data shares in canvas tools.